### PR TITLE
2.x docs migration

### PR DESCRIPTION
**Pull request by RaasAhsan**
[ ] Delete microsite configuration

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -58,12 +47,12 @@ val SilencerVersion = "1.7.1"
 
 replaceCommandAlias(
   "ci",
-  "; project /; headerCheck ;scalafmtSbtCheck ;scalafmtCheckAll; clean; testIfRelevant; mimaReportBinaryIssuesIfRelevant; doc"
+  "; project /; headerCheck ;scalafmtSbtCheck ;scalafmtCheckAll; clean; testIfRelevant; mimaReportBinaryIssuesIfRelevant; doc; docs/mdoc"
```
Comment (djspiewak) [inline build.sbt]: Shouldn't we still run mdoc on build to ensure we don't break anything?

Comment (RaasAhsan) [reply build.sbt]: `docs/mdoc` does that right? and `sbt ci` is run on every build

Comment (djspiewak) [reply build.sbt]: Oh derp I read the diff backwards. Nvm!

**Comment by djspiewak** [comment]
@RaasAhsan When you get a chance can you bring this up to date with `HEAD`?

**Comment by RaasAhsan** [comment]
```
[error] (run-main-0) java.lang.ClassNotFoundException: mdoc.Main
[error] java.lang.ClassNotFoundException: mdoc.Main
[error] 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
```
What did i mess up

**Comment by djspiewak** [comment]
…what

**Comment by RaasAhsan** [comment]
It happens locally for Scala `3.0.0-M2` but not for `2.12` or `2.13`

**Comment by djspiewak** [comment]
Oh, I think we should probably turn off mdoc for anything that isn't 2.13, at least for now. I *think* that's what other projects are doing at present?

**Comment by RaasAhsan** [comment]
Seems like it's what we're doing on series/3.x: https://github.com/typelevel/cats-effect/blob/series/3.x/build.sbt#L69

**Pull request merged**